components/PageButtons/src/index.css

```diff
@@ -0,0 +1,67 @@
+.denhaag-page-buttons {
+  display: flex;
+  flex-direction: column;
+  margin-block-start: var(--denhaag-page-buttons-margin-block-start);
+  margin-block-end: var(--denhaag-page-buttons-margin-block-end);
+}
+
+.denhaag-page-buttons__outline-dark-action {
+  border-style: solid;
+  background-color: var(--denhaag-page-buttons-outline-dark-action-background-color);
+  border: var(--denhaag-button-border-width) solid var(--denhaag-page-buttons-outline-dark-action-border-color);
+  color: var(--denhaag-page-buttons-outline-dark-action-color);
+  min-width: var(--denhaag-page-buttons-outline-dark-action-min-width);
+  margin-inline-end: var(--denhaag-page-buttons-outline-dark-action-margin-inline-end);
+  align-self: baseline;
+  margin-block-end: var(--denhaag-page-buttons-outline-dark-action-margin-block-end);
+}
+
+.denhaag-page-buttons__outline-dark-action.denhaag-button--hover,
+.denhaag-page-buttons__outline-dark-action:hover {
+  background-color: var(--denhaag-button-secondary-action-hover-background-color);
+  border-color: var(--denhaag-button-secondary-action-hover-border-color);
+  color: var(--denhaag-button-secondary-action-hover-color);
+}
+
+.denhaag-page-buttons__outline-dark-action__icon {
+  margin-inline-end: var(--denhaag-page-buttons-icon-margin-inline-end);
+  max-height: var(--denhaag-page-buttons-icon-max-height);
+  max-width: var(--denhaag-page-buttons-icon-max-width);
+}
+
+.denhaag-share-button__options {
+  display: none;
+}
+
+.denhaag-page-buttons--open .denhaag-share-button__options {
+  display: flex;
+}
+
+.denhaag-share-button__option {
+  padding-block-start: 0;
+  padding-block-end: 0;
+  padding-inline-start: 0;
+  padding-inline-end: 0;
```

Comment:
I think this can be shortened to `padding: 0`

Reply:
Lint don't want that?: 
```
 49:3  ✖  Unexpected property "padding"   property-disallowed-list
 ```